cves/CVE-2016-8747.yml

```diff
@@ -141,7 +141,7 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this
     section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: false
```

Comment:
Were there any commits at all in the same subsystem between the VCC and fix?

Reply:
Upon initial research: no. I will double check.